webapp/app/partials/perf/perfctrl.html

```diff
@@ -16,6 +16,7 @@
             <span class="sr-only">Remove series</span>
           </button>
           <table>
+            <input title="Show/Hide series" type="checkbox" id="showHideCheck" ng-click="hideSeries(series.signature)" checked>
```

Comment:
I think we should try to align this element in the bottom right corner (floating), as is done in graphs.mozilla.org. As it is, it really distracts from the series data information which is what we want to call attention to. Could you adjust the CSS to make this happen? If you need help, feel free to ask either myself or @tojonmz